api/cpp/cmake/SlintMacro.cmake

```diff
@@ -6,6 +6,28 @@ function(SLINT_TARGET_SOURCES target)
         get_filename_component(_SLINT_BASE_NAME ${it} NAME_WE)
         get_filename_component(_SLINT_ABSOLUTE ${it} REALPATH BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
         get_property(_SLINT_STYLE GLOBAL PROPERTY SLINT_STYLE)
+        get_target_property(_SLINT_EMBED_RESOURCES ${target} SLINT_EMBED_RESOURCES)
```

Comment:
Hm, this means that if you write

```cmake
add_target(foo ...)
slint_target_sources(foo ...)
set_property(TARGET foo PROPERTY SLINT_EMBED_RESOURCES true)
```

it won't work.

Perhaps this could be implemented as a generator expression instead?